### PR TITLE
[KOGITO-2037] Adjust process-quarkus-example example for Kogito operator

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

https://issues.redhat.com/browse/KOGITO-2037

Should be merged together with https://github.com/kiegroup/kogito-examples/pull/253

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by sutaakar** [comment]
One test was failing on Jenkins, locally passed, rerunning build.

**Comment by radtriste** [comment]
@sutaakar let me know once ok

**Comment by sutaakar** [comment]
@radtriste Rerun helped, can be merged (together with the other PR)

File: test/steps/utils.go [hunk 1]
```diff
@@ -121,8 +121,6 @@ func parseConfigRow(row *messages.PickleStepArgument_PickleTable_PickleTableRow,
 		if events {
 			*profilesPtr = append(*profilesPtr, "events")
 			kogitoApp.Spec.EnableEvents = true
-			kogitoApp.Spec.KogitoServiceSpec.AddEnvironmentVariable("MP_MESSAGING_OUTGOING_KOGITO_PROCESSINSTANCES_EVENTS_BOOTSTRAP_SERVERS", "")
-			kogitoApp.Spec.KogitoServiceSpec.AddEnvironmentVariable("MP_MESSAGING_OUTGOING_KOGITO_USERTASKINSTANCES_EVENTS_BOOTSTRAP_SERVERS", "")
```
Comment (MarianMacik) [inline test/steps/utils.go:125]: Won't this affect other examples? Or they are already OK with that?

BTW these variables were up to now empty as it was just for operator to know that it should inject the values to these variables?

Comment (sutaakar) [reply test/steps/utils.go:125]: > Won't this affect other examples? Or they are already OK with that?

Other examples already switched to `kafka.bootstrap.servers` approach (which is injected by Kogito operator automatically). The process-quarkus-example was the last one using the old approach.

> BTW these variables were up to now empty as it was just for operator to know that it should inject the values to these variables?

Yes.

**Comment by radtriste** [comment]
waiting for https://github.com/kiegroup/kogito-examples/pull/253 to be merged

**Pull request merged**